test/suites/integration/nodeclass_test.go

```diff
@@ -109,4 +109,12 @@ var _ = Describe("NodeClass IAM Permissions", func() {
 			g.Expect(nodeClass.StatusConditions().Get(v1.ConditionTypeValidationSucceeded).IsTrue()).To(BeTrue())
 		}).Should(Succeed())
 	})
+	It("should bypass and succeed EC2NodeClass validation when validation is disabled", func() {
+		env.ExpectSettingsOverridden(corev1.EnvVar{Name: "disableAuthValidation", Value: "true"})
+		env.ExpectCreated(nodeClass)
```

Comment:
I think we make a policy update that would normally fail to validate that we always pass the validation? or at least check the reason